sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/SarifParserCallbackImpl.java

```diff
@@ -248,11 +248,15 @@ public void onRule(String ruleId, @Nullable String shortDescription, @Nullable S
       .ruleId(ruleId)
       .severity(mapSeverity(defaultLevel))
       .name(shortDescription != null ? shortDescription : ruleId)
-      .description(fullDescription != null ? fullDescription : null)
+      .description(fullDescription)
       .type(ruleType)
       .save();
   }
 
+  private boolean shouldCreateExternalIssue(String ruleId) {
+    return !ignoreThirdPartyIssues && !ruleId.matches("^S\\d{3,5}$");
```

Comment:
We decided to go with the simplest check - if the RuleId matches our own format we do not report it as external.

Reply:
Are we confident enough that other analyzers don't use this pattern? If yes, then LGTM ;)